src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/config.jelly

```diff
@@ -30,7 +34,11 @@
         <c:select/>
     </f:entry>
 
-    <f:entry field="aid" title="Analysis ID Override">
+    <f:invisibleEntry field="aid" title="Analysis ID Override" >
```

Comment:
Alessandro has made this UI field invisible, so users should not see it or use it in freestyle projects.  In the context of the freestyle project, calling this an "override" seemed reasonable to me (although GT did not invent the language).  In any case, we will still remove "override" everywhere in case this field is somehow made visible despite our efforts to hide it.